connector/rest/src/main/java/it/pagopa/selfcare/mscore/connector/rest/UserRegistryConnectorImpl.java

```diff
@@ -31,10 +32,10 @@ public UserRegistryConnectorImpl(UserRegistryRestClient restClient, UserMapperCl
     }
 
     @Override
-    public User getUserByInternalId(String userId) {
+    public User getUserByInternalId(String userId, boolean withFiscalCode) {
```

Comment:
it might be better to create 2 methods instead using a boolean...for example getUserByInternalId and getUserByInternalIdWithFiscalCode